rq_scheduler/scheduler.py

```diff
@@ -109,7 +109,7 @@ def stop(signum, frame):
 
     def _create_job(self, func, args=None, kwargs=None, commit=True,
                     result_ttl=None, ttl=None, id=None, description=None,
-                    queue_name=None, timeout=None, meta=None):
+                    depends_on=None, queue_name=None, timeout=None, meta=None):
```

Comment:
Can you please move `depends_on` to the end of the argument list? Don't want to break codes that use positional arguments.

Reply:
aye